<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_usuarios</class>
 <widget class="QWidget" name="form_usuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usuarios de Active Directory</string>
  </property>
  <widget class="QLineEdit" name="text_usuario">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>11</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>0</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>47</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Usuario&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_nombre">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>41</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MARIA PILAR FERNANDEZ DE LAS HERAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>41</y>
     <width>49</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nombre&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>72</y>
     <width>39</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Correo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_correo">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>72</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_estado">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>143</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>205</y>
     <width>40</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Creada</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_creada">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>205</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>174</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefono</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>143</y>
     <width>39</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_telefono">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>174</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>236</y>
     <width>42</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Caduca</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_cuenta_caduca">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>236</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>104</y>
     <width>37</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha 
correo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_fecha_correo">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>108</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="title">
    <string>Contraseña</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>124</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Clave caduca</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>62</y>
      <width>120</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Ultima modificación</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_modif_clave">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>62</y>
      <width>108</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_cambio_clave">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>31</y>
      <width>108</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_clave_caduca">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>124</y>
      <width>108</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>31</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Ultimo cambio clave</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>93</y>
      <width>120</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Intentos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_intentos">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>93</y>
      <width>108</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Boton_buscar">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>BUSCAR</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_usuarios">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>256</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(32, 74, 135);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_logon">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>41</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>41</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Nº Logon</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_ulti_login">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>72</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>72</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Ultimo login</string>
   </property>
  </widget>
  <zorder>text_usuario</zorder>
  <zorder>label</zorder>
  <zorder>text_nombre</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>text_correo</zorder>
  <zorder>text_estado</zorder>
  <zorder>label_4</zorder>
  <zorder>text_creada</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>text_telefono</zorder>
  <zorder>label_13</zorder>
  <zorder>text_cuenta_caduca</zorder>
  <zorder>label_14</zorder>
  <zorder>text_fecha_correo</zorder>
  <zorder>groupBox</zorder>
  <zorder>Boton_buscar</zorder>
  <zorder>text_logon</zorder>
  <zorder>label_12</zorder>
  <zorder>text_ulti_login</zorder>
  <zorder>label_9</zorder>
  <zorder>list_usuarios</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
